### DISCUSSION TITLE
ScriptReference/Physics.OverlapCapsuleNonAlloc

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#543] [Notes]
# ScriptReference/Physics.OverlapCapsuleNonAlloc

https://docs.unity3d.com/ScriptReference/Physics.OverlapCapsuleNonAlloc.html

**Comment by Flewd** [comment]
point0 is the top of the capsule
point1 is the bottom of the capsule
